examples/dataflow-data-generator/data-generator-pipeline/tests/test_fix_record_for_avro.py

```diff
@@ -0,0 +1,76 @@
+# Copyright 2018 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#            http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import avro.schema
+import json
+import logging
+import unittest
+
+from data_generator.AvroUtil import fix_record_for_avro
+
+class TestAvroFixer(unittest.TestCase):
+    def test_fix_record_for_avro(self):
+
+        avro_schema = avro.schema.parse(
+            json.dumps({
+                 u'type': u'record',
+                 u'name': u'AthleticsWorldRecords',
+                 u'fields': [
+                    {u'name': u'birthday', 
+                    u'type': [u'null', 
+                        {u'logicalType': u'date', u'type': u'int'}]},
+                    {u'name': u'athlete', u'type':'string'},
+                    {u'name': u'race_start_time', 
+                    u'type': [u'null', 
+                        {u'logicalType': u'time-micros', u'type': u'long'}]},
+                    {u'name':u'race_start_datetime',
+                    u'type': [u'null', 
+                        {u'logicalType': u'timestamp-millis', u'type': u'long'}]},
+                    {u'name':u'race_end_timestamp', 
+                    u'type': [u'null', 
+                        {u'logicalType': u'timestamp-micros', u'type': u'long'}]},
+                    {u'name':u'race_distance_m', u'type':'int'},
+                    {u'name':u'time_seconds', u'type':'float'},
+                    {u'name':u'is_world_record', u'type':'boolean'}
```

Comment:
Do we need unicode Strings?

Reply:
I use unicode strings because that's what the `json` library does by default.